src/firmware/boards/legacy_robot_stm32f4/io/receive.c

```diff
@@ -367,28 +348,28 @@ void handle_camera_packet(uint8_t *dma_buffer, uint8_t buffer_position)
 
 void handle_other_packet(uint8_t *dma_buffer, size_t frame_length)
 {
-    // printf("got a message with purpose: %i", dma_buffer[MESSAGE_PURPOSE_ADDR]);
-    // printf("var index: %i", dma_buffer[MESSAGE_PURPOSE_ADDR + 1]);
-    // printf("value: %i", dma_buffer[MESSAGE_PURPOSE_ADDR + 2]);
-    switch (dma_buffer[MESSAGE_PURPOSE_ADDR])
+    // printf("got a message with purpose: %i", dma_buffer[MESSAGE_PURPOSE_INDEX]);
+    // printf("var index: %i", dma_buffer[MESSAGE_PURPOSE_INDEX + 1]);
```

Comment:
Same comment about code for debugging?

Reply:
So these aren't my debug statements, they were there already, they just got changed because I renamed a variable that is used in them. In the interests of limiting scope creep I'm going to say we should just leave them for now.